apps/chef_objects/src/chef_object_base.erl

```diff
@@ -60,78 +58,6 @@
 -compile([export_all]).
 -endif.
 
--spec set_created(Object :: chef_object() |
```

Comment:
Just a note that coverage compile and run showed these paths completely unused, as did a follow-up grep through code. These are all handled in the individual [oc] chef object modules.